apps/web/pages/signup.tsx

```diff
@@ -130,10 +131,11 @@ export default function Signup({ prepopulateFormValues, token, orgSlug }: Signup
                   <TextField
                     addOnLeading={
                       orgSlug
-                        ? getOrgFullDomain(orgSlug, { protocol: false })
+                        ? getOrgFullDomain(orgSlug, { protocol: true })
                         : `${process.env.NEXT_PUBLIC_WEBSITE_URL}/`
                     }
                     {...register("username")}
+                    disabled={!!orgSlug}
```

Comment:
We don't want to let the user change their designated username based on their email address